RIOT-OS/RIOT
# makefiles/info-global.inc.mk: bug: honor DEFAULT_MODULES

### Contribution description

The parsing for info-boards-supported was completely ignoring default
modules.

### Testing procedure

Review that both lines are done in `Makefile.include`.
Murdock should compile everything without error.

I will post a diff using dependencies resolution output when I have it.



When running the same procedure as in https://github.com/RIOT-OS/RIOT/pull/12014
And comparing the difference of the aggregated `info-boards-supported` debug output I get:

```
diff -y --suppress-common-lines build/deps/ref/deps_info-boards-supported build/deps/2_default_modules/deps_info-boards-supported  | grep  -v USEMODULE | sort -u
DEFAULT_MODULE =                                              | DEFAULT_MODULE = board cpu core core_msg sys auto_init
DISABLE_MODULE =                                              | DISABLE_MODULE = auto_init
```

And we can see that `USEMODULE` now has values from the `DEFAULT_MODULE`:


```
diff -y --suppress-common-lines build/deps/ref/deps_info-boards-supported build/deps/2_default_modules/deps_info-boards-supported  | head -n 10
DEFAULT_MODULE =                                              | DEFAULT_MODULE = board cpu core core_msg sys auto_init
DISABLE_MODULE =                                              | DISABLE_MODULE = auto_init
USEMODULE = checksum periph_common periph_gpio periph_pm riot | USEMODULE = board checksum core core_msg cpu periph_common pe
DEFAULT_MODULE =                                              | DEFAULT_MODULE = board cpu core core_msg sys auto_init
DISABLE_MODULE =                                              | DISABLE_MODULE = auto_init
USEMODULE = checksum periph_common periph_gpio periph_pm riot | USEMODULE = board checksum core core_msg cpu periph_common pe
DEFAULT_MODULE =                                              | DEFAULT_MODULE = board cpu core core_msg sys auto_init
DISABLE_MODULE =                                              | DISABLE_MODULE = auto_init
USEMODULE = boards_common_arduino-atmega boards_common_atmega | USEMODULE = board boards_common_arduino-atmega boards_common_
DEFAULT_MODULE =                                              | DEFAULT_MODULE = board cpu core core_msg sys auto_init
```

### Issues/PRs references

This is based on

* makefiles/info-global.inc.mk: bug: fix DISABLE_MODULE_GLOBAL value #12010 
* make: add targets to debug dependencies variables #12004 for debugging
